src/mixins/nft.js

```diff
@@ -320,11 +320,25 @@ export default {
       }
     },
     async updateNFTClassMetadata() {
-      await this.fetchNFTMetadata(this.classId);
+      try {
+        await this.fetchNFTMetadata(this.classId);
+      } catch (error) {
+        if (error.response?.status !== 404) {
+          // eslint-disable-next-line no-console
+          console.error(error);
```

Comment:
please use JSON.stringify since axios error is multiline

Reply:
Updated